app/gui/view/src/root.rs

```diff
@@ -0,0 +1,152 @@
+//! Root View of the IDE.
+//!
+//! The main entry point to the IDE which can display either Welcome Screen or Project View.
+//! Initially displays Welcome Screen. As soon as user selects a project to open, Root View
+//! switches state to Project View and stays in this state till the application quit.
```

Comment:
This is not the desired behaviour. We want to be able to open the Welcome Screen during app usage on demand.

Reply:
Added new FRP input `switch_view_to_welcome_screen`.